phonebook/PhoneDirectory.java

```diff
@@ -0,0 +1,36 @@
+package phonebook;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+
+public class PhoneDirectory{
+    private Map<String, List<Record>> phonebook;
```

Comment:
Как уже писал, в этой задание не включает использование словарей.
```suggestion
    private List<Record> phonebook;
```